src/cloudflare/internal/ai-api.ts

```diff
@@ -128,39 +128,27 @@ function findReadableStreamKeys(
 }
 
 export class Ai {
-  // TODO(soon): Can we use the # syntax here?
-  // eslint-disable-next-line no-restricted-syntax
-  private readonly fetcher: Fetcher;
-
-  /*
-   * @deprecated this option is deprecated, do not use this
-   */
-  // TODO(soon): Can we use the # syntax here?
-  // @ts-expect-error this option is deprecated, do not use this
-  // eslint-disable-next-line no-restricted-syntax
-  private logs: Array<string> = [];
```

Comment:
logs is being removed, is that because it was already deprecated?

Reply:
I've reverted these changes. Will open a new PR for discussion.